packages/components/src/components/hds/form/radio-card/index.hbs

```diff
@@ -18,8 +18,8 @@
       name={{@name}}
       checked={{@checked}}
       disabled={{@disabled}}
-      aria-describedby={{this.ariaDescribedBy}}
+      aria-describedby={{@extraAriaDescribedBy}}
```

Comment:
since `this.ariaDescribedBy` wasn't updated via children, we can just use `@extraAriaDescribedBy` for this component